webapp/app/commands.py

```diff
@@ -117,7 +117,6 @@ def _revoke_permission_and_delete_users(users_to_delete, success_message):
                 'elster_request_id': user_to_delete.elster_request_id}
         else:
             form_data = {
-                'idnr': user_to_delete.idnr_hashed,
```

Comment:
```python
PRODUCTION_TEST_IDNR = '04452397687'
        if user_to_delete.idnr_hashed == global_salt_hash().hash(PRODUCTION_TEST_IDNR):
            # Ensure that testmerker is used to actually revoke the permission
            form_data = {
                'idnr': PRODUCTION_TEST_IDNR,
                 'elster_request_id': user_to_delete.elster_request_id}
```
Unfortunately, I am not able to comment this part above, but I think we could get rid of that now because we do check the id number of the elster_request_id in the erica. Furthermore, this is only for one TEST_IDNR while we use more than this one on production. wdyt?

Reply:
Yeah i wasn't sure about that but if you say elster check the id number of the passed elster_request_id, then yes. I will remove it and then merge and deploy on prod and check the cron jobs. Thanks!